config.md

```diff
@@ -27,13 +27,15 @@ Each container has exactly one *root filesystem*, specified in the *root* object
 
 * **`path`** (string, required) Specifies the path to the root filesystem for the container. A directory MUST exist at the path declared by the field.
 * **`readonly`** (bool, optional) If true then the root filesystem MUST be read-only inside the container. Defaults to false.
+* **`autoremove`** (bool, optional) If true then the container will be automatically deleted once the PID namespace is deleted. This removes the need for an explicit call to the `delete` operation. Defaults to false.
```

Comment:
> once the PID namespace is deleted

Isn't this too technical detailed? What if a container doesn't have it's own PID namespace?
Is we call it `processes of the container are exited` sufficient enough?